packages/client/src/types/priceType.types.ts

```diff
@@ -0,0 +1,5 @@
+export enum PriceTypeEnum {
```

Comment:
Hmm, I'd suggest adding this on the `price.types.ts` file, it's where all price-related things are aggregated, instead of having a separate file for each price thing 🤔

Reply:
Done